benchmarks/src/main/java/org/elasticsearch/benchmark/index/mapper/MapperServiceFactory.java

```diff
@@ -71,7 +71,8 @@ public static MapperService create(String mappings) {
                 public <T> T compile(Script script, ScriptContext<T> scriptContext) {
                     throw new UnsupportedOperationException();
                 }
-            }
+            },
+            /*()->DocumentParsingObserver.EMPTY_INSTANCE*/null // we don't want to export this package every
```

Comment:
debugging comment.

Reply:
fixed and replied here https://github.com/elastic/elasticsearch/pull/97961#discussion_r1279536506